addon/road_base.py

```diff
@@ -336,39 +357,37 @@ def compare_boundaries_with_s(self, s, s_boundaries_next):
 
         return smaller, idx_sorted[0]
 
-    def get_road_vertices_edges_faces(self, strips, road_sample_points):
-        '''
-            Return blender compatible vertices, edges and faces concatenated and
-            calculated from road sample points.
-        '''
+    # generate mesh from samplepoints
```

Comment:
Should use triple ticks to document function like it was before.

Reply:
done
![image](https://user-images.githubusercontent.com/11625259/149607489-01001405-4247-4a45-94ec-da11f0ca738b.png)